SmartAPIRegistration.md

```diff
@@ -0,0 +1,55 @@
+# SmartAPI Registrations
+
+
+## Background
+
+The SmartAPI registry is Translator's discovery registry.  Translator tools create a yaml registration that can be queried through the SmartAPI registry. 
+
+This document defines the Translator requirements for such registrations.
+
+Translator components are registered using OpenAPI metadata as described [here](https://smart-api.info/guide).   In addition to those general requirements, specific additions are required for Translator components.
+
+## Requirements
+
+### Translator tag
+
+"translator" must be included as a top-level OpenAPI tag.
+
+### x-translator element
+
+The registration must contain a compliant x-translator element as described [here](https://github.com/NCATSTranslator/translator_extensions/tree/production/x-translator).  Among other things, the x-translator element defines the infores identifier for this component: an identifier that is common across environments such as dev, test, and production.
+
+### Trapi tag
+
+If the component implements a TRAPI interface, then "trapi" must be included as a top-level OpenAPI tag.
+
+### x-trapi element
+
+If the component implements a TRAPI interface, then the registration must contain a compliant x-trapi element as described [here](https://github.com/NCATSTranslator/translator_extensions/tree/production/x-trapi).
+
+### Uptime checking
+
+Registrations must enable uptime checking as described [here](https://smart-api.info/faq#api-monitor).
```

Comment:
here is a more detailed uptime documentation you can link to instead: https://github.com/SmartAPI/smartAPI/blob/master/docs/uptime.md